resources/conf.rb

```diff
@@ -20,4 +20,4 @@
 default_action :create
 
 attribute :config,
-          kind_of: Array
+          kind_of: Hash
```

Comment:
I'm leery about whether this potentially creates an incompatibility? Also, not sure why this is an Array, since the config attribute is a hash, and I don't know what else someone would put here. :)